src/testers.cpp

```diff
@@ -254,22 +254,33 @@ bool ValueTester::Test(const InternalValue& baseVal, RenderContext& context)
     {
         bool isConverted = false;
         auto seq = GetArgumentValue("seq", context);
-        ListAdapter values = ConvertToList(seq, InternalValue(), isConverted);
-
-        if (!isConverted)
-            return false;
-
-        auto equalComparator = [&baseVal](auto& val) {
-            InternalValue cmpRes;
-
-            cmpRes = Apply2<visitors::BinaryMathOperation>(val, baseVal, BinaryExpression::LogicalEq);
-
-            return ConvertToBool(cmpRes);
-        };
-
-        auto p = std::find_if(values.begin(), values.end(), equalComparator);
-        result = p != values.end();
-
+        auto seqKind = Apply<ValueKindGetter>(seq); 
+        if (seqKind == ValueKind::List) {
+            ListAdapter values = ConvertToList(seq, InternalValue(), isConverted);
+
+            if (!isConverted)
+                return false;
+
+            auto equalComparator = [&baseVal](auto& val) {
+                InternalValue cmpRes;
+                cmpRes = Apply2<visitors::BinaryMathOperation>(val, baseVal, BinaryExpression::LogicalEq);
+                return ConvertToBool(cmpRes);
+            };
+
+            auto p = std::find_if(values.begin(), values.end(), equalComparator);
+            result = p != values.end();
+        } else if (seqKind == ValueKind::String) {
+            result = ApplyStringConverter(baseVal, [&](const auto& srcStr) {
+                    std::decay_t<decltype(srcStr)> emptyStrView;
+                    using CharT = typename decltype(emptyStrView)::value_type;
+                    std::basic_string<CharT> emptyStr;
+
+                    auto substring = sv_to_string(srcStr);
+                    auto seq = GetAsSameString(srcStr, this->GetArgumentValue("seq", context)).value_or(emptyStr);
```

Comment:
This feels too verbose but I haven't found a simpler way to parse arguments.
I used this place as a reference:
https://github.com/jinja2cpp/Jinja2Cpp/blob/171b5bd970579bb387ad49703fad0313eaf0e1f3/src/string_converter_filter.cpp#L247-L253